haystack/translator/transformers.py

```diff
@@ -73,6 +73,12 @@ def translate(
     ) -> Union[str, List[Document], List[str], List[Dict[str, Any]]]:
         """
         Run the actual translation. You can supply a query or a list of documents. Whatever is supplied will be translated.
+        :param query: The query string to translate
+        :type query: str
```

Comment:
Do we really want to keep the type annotations in the docstring? We have them now already in the method signatures and most other Haystack methods don't have them in the docstring anymore. Are there any advantages of having them for the "API reference" in the docs?

Reply:
Ok let's stick to type annotations in method signatures, not in docstrings